SMPEmu.ino

```diff
@@ -220,16 +220,7 @@ void setup() {
 }
 
 void loop() {
-   digitalWrite(13, LOW);
-   
-  while(digitalRead(SELECT_PIN)) 
```

Comment:
Not sure if you need to remove those according to the datasheet, but to eliminate race conditions this might be helpful